refs/files-backend.c

```diff
@@ -3473,6 +3494,7 @@ static int files_fsck_refs(struct ref_store *ref_store,
 	struct fsck_refs_info info;
```

Comment:
Why do we create this here and pass it to every `fsck_refs_fns`? We don't seem to add any info to it, nor do we use the information added to it. Makes sense to remove it from here and initialize it inside each of the `fsck_refs_fns`

Reply:
Well, I have a reason to do this. The reason why I initialize this in `files_fsck_refs` is because we only need to initialize this once. So I made it a state here.

But I agree with you we should not add this variable into the parameter which makes the code messy here.

I will improve this. Thanks